specification/columns/chargeclass.md

```diff
@@ -1,8 +1,8 @@
 # Charge Class
 
-A Charge Class indicates whether the row represents a regular charge or a correction to one or more previous charges. Charge Class is commonly used to differentiate refunds from regularly incurred charges.
+A Charge Class indicates whether the row represents a correction to one or more [*charges*](#glossary:charge) that have been invoiced in previous billing periods. Charge Class is commonly used to differentiate refunds from regularly incurred charges.
 
-The ChargeClass column MUST be present and MUST NOT be null. This column is of type String and MUST be one of the allowed values.
+The ChargeClass column MUST be present in the billing data. This column MUST be of type String and MUST be set to "Correction" in case of a correction to one or more charges already invoiced in previous billing periods. Otherwise, it MUST be null.

```

Comment:
Minor phrasing nit and making the last sentence a bit more explicit to hopefully minimize ambiguity.
```suggestion
The ChargeClass column MUST be present in the billing data. This column MUST be of type String and MUST be "Correction" when the row represents a correction to one or more charges invoiced in previous billing periods. ChargeClass MUST be null when it is not a correction or when it is a correction within the current billing period.
```

Reply:
suggested change applied.